MAUI/Calendar/events.md

```diff
@@ -0,0 +1,182 @@
+The Calendar has the events `ViewChanged`, `SelectionChanged`, `Tapped`, `DoubleTapped` and `LongPressed` for notifying after user interactions in [.NET MAUI Calendar]. 
+
+## ViewChanged
+
+The [ViewChanged] event is used to notify when the current view of calendar is changed while the view is swiped to previous or next view, and when the calendar view is switched to another calendar view.
+
+* `Sender`: This contains the `SfCalendar` object.
+
+* `ViewChanged`: The Calendar current view visible dates are available in the [CalendarViewChangedEventArgs] when the calendar visible dates or view is changed.
+
+    * [NewVisibleDates] : Returns the new visible date range of the view.
+    * [OldVisibleDates] : Returns the old visible date range of the view.
+    * [NewView] : Returns the new calendar view. 
+    * [OldView] : Returns the old calendar view.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                      ViewChanged="OnCalendarViewChanged">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+
+this.Calendar.ViewChanged += OnCalendarViewChanged;
+private void OnCalendarViewChanged(object sender, CalendarViewChangedEventArgs e)
+{
+    var oldVisibleDates = e.OldVisibleDates;
+    var newVisibleDates = e.NewVisibleDates;
+    var oldCalendarView = e.OldView;
+    var newCalendarView = e.NewView;
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N>
+Whenever a calendar view is changed, the `SelectableDayPredicate` function must be called to decide whether the cell is selectable or not in the `SfCalendar.`
+
+## SelectionChanged
+
+The [SelectionChanged] event is used to notify when the cell selection is changed onto the view in the [SfCalendar].
+
+* `Sender`: This contains the `SfCalendar` object.
+
+* `SelectionChanged`: The selection is changed when the selection-changed action is performed on the element available in the [CalendarSelectionChangedEventArgs].
+
+    * [NewValue] : Returns the new selected date.
+    * [OldValue] : Returns the old selected date.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                      SelectionChanged="OnCalendarSelectionChanged">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+
+this.Calendar.SelectionChanged += this.OnCalendarSelectionChanged;
+
+private void OnCalendarSelectionChanged(object sender,  private void OnCalendarSelectionChanged(object sender, CalendarSelectionChangedEventArgs e)
+    {
+        var oldDateTime = e.OldValue;
+        var newDateTime = e.NewValue;
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+N>
+The `Tapped` event is triggered first, followed by 'selection-changed' event will be performed.
+
+## Tapped
+
+A [Tapped] event is triggered, each time a calendar view is tapped.
+Below is a list of the arguments:
```

Comment:
The following is a list of the arguments:

Reply:
Tapped content changed